ui/src/views/Documentation/Reference/ApiReference.jsx

```diff
@@ -9,6 +9,29 @@ import HeaderWithAnchor from '../components/HeaderWithAnchor';
 import Anchor from '../components/Anchor';
 import findRefDoc from '../../../utils/findRefDoc';
 
+class GroupEntry extends Component {
+  render() {
+    const { group, serviceName } = this.props;
+    const [groupName, listOfEntries] = group;
+
+    return (
+      <div>
+        <br />
+        <Typography> {groupName} </Typography>
+        <br />
+        {listOfEntries.map(entry => (
+          <Entry
+            key={`${entry.name}-${entry.query}`}
+            type="function"
+            entry={entry}
+            serviceName={serviceName}
+          />
+        ))}
+      </div>
+    );
+  }
+}
```

Comment:
We can inline all of this inside the render method of ApiReference. No need to create a new class.

Reply:
Applied the changes!